<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конструктор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(193, 226, 214);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mainwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(175, 206, 195);</string>
    </property>
    <widget class="QGroupBox" name="settings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>311</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(230, 186, 191);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="width_lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="height_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="height_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 186, 191);</string>
      </property>
      <property name="text">
       <string>Размер изделия</string>
      </property>
     </widget>
     <widget class="QLabel" name="cloth_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 186, 191);</string>
      </property>
      <property name="text">
       <string>Выбор ткани</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cloth_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="furnitura_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(230, 186, 191);</string>
      </property>
      <property name="text">
       <string>Фурнитура</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="furnitura_tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>291</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);

alternate-background-color: #FFFCD6;
 gridline-color: #B0BEC5;

</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cloth_add_pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="furniture_add_pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>360</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Название изделия</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>113</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_edit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>мм</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rotate_left">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Повернуть</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rotate_right">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_panel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Панель инструментов</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_constructor_pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>670</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(230, 186, 191);</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>670</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(230, 186, 191);</string>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_icon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="product_name_label">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(181, 213, 202, 60);</string>
    </property>
    <property name="text">
     <string>Конструктор изделий</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_constructor" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>880</width>
      <height>660</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255); border: 5px solid rgb(224, 169, 175);</string>
    </property>
   </widget>
  </widget>
  <action name="open_action">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
